### PR TITLE
11378: new Hadoop Hive Data Source on `#/data_sources` page

### PR DESCRIPTION
**Pull request by kevin-alpine**
alpine chorus

@jleealpine @midnighteuler @AndrewLngdn please comment?  I have not addressed the tests yet, I will do that now.

https://alpine.atlassian.net/browse/DEV-11378

[new functionality, backbone, M, migration] DEV-11378 User can create a HIVE data source within Chorus ... Chester showed me how he can already connect to a "Hadoop Hive" datasource by adding certain custom "configuration parameters" ... basically this commit formalizes his approach by adding a new DataSource type (in the dialog only) and then, modifying the existing HdfsDataSource with a boolean and a new string field ... this is then hacked into the HdfsDataSource#hdfs_pairs and the QueryService module, which looks to me like the place where this info is passed along to Hadoop.

### CODE REVIEW

File: app/models/hdfs_data_source.rb [hunk 1]
```diff
@@ -68,7 +68,18 @@ def supports_work_flows
...
+    if connection_parameters
+      connection_parameters.each { |hsh| pairs.push com.emc.greenplum.hadoop.plugins.HdfsPair.new(hsh['key'], hsh['value']) }
+    end
+
+    if is_hive
+      pairs.push com.emc.greenplum.hadoop.plugins.HdfsPair.new('is_hive', is_hive)
+      pairs.push com.emc.greenplum.hadoop.plugins.HdfsPair.new('hive.metastore.uris', hive_metastore_location)
+    end
+
+    pairs
```
Comment (kevin-alpine) [inline app/models/hdfs_data_source.rb]: @jleealpine: Can you comment on this change specifically?  I'm making a wild guess that this is the spot where I can pass along Chester's configuration options to Hadoop.

**Comment by mmilano** [comment]
Please change the entry in the list from "Hadoop Hive Data Source" to simply "Hadoop Hive".

This also applies to anywhere that the data source type appears in the interface.

**Comment by kevin-alpine** [comment]
@mmilano -- I renamed "Hadoop Hive Data Source" to "Hadoop Hive"

**Comment by jleealpine** [comment]
Changes look good

**Comment by kevin-alpine** [comment]
Ok, I'm merging this in.

**Pull request merged**